core/src/impl/Kokkos_Core.cpp

```diff
@@ -824,8 +824,14 @@ instead of hyphens). For example, to disable warning messages, you can either
 specify --kokkos-disable-warnings or set the KOKKOS_DISABLE_WARNINGS
 environment variable to yes.
 
-Join us on Slack, visit https://kokkosteam.slack.com
-Report bugs to https://github.com/kokkos/kokkos/issues
+--------------------------------------------------------------------------------
+For support, documentation, and more information about Kokkos,
+visit the official website: https://kokkos.org
+
+If you use Kokkos in your work, please cite
+  https://doi.ieeecomputersociety.org/10.1109/TPDS.2021.3097283
```

Comment:
We already have a CITATION.cff file that is parsed for the "Cite this repository" link. Maybe, we should just point users there so we don't gave to manage multiple locations.

Reply:
What URL would we share?
How do we ensure it is permanent?

Reply:
Something like
```suggestion
If you use Kokkos in your work, please use the references pointed to in "Cite this repository" or https://github.com/kokkos/kokkos/blob/develop/CITATION.cff.
```